man/journald.conf.xml

```diff
@@ -363,7 +363,38 @@
         <listitem><para>Takes a boolean value. If enabled (the
         default), journal reads <filename>/dev/kmsg</filename>
         messages generated by the kernel.</para></listitem>
-      </varlistentry>
+         </varlistentry>
+
+         <varlistentry>
+           <term><varname>FacilitiesStore=</varname></term>
+
+        <listitem><para>Controls the log facility of messages
+        that are stored on disk, forwarded to syslog, kmsg, the
+        console or wall (if that is enabled, see above). As argument,
+        takes one or more space separated facility names <option>kern</option>,
+        <option>user</option>, <option>mail</option>,
+        <option>daemon</option>, <option>auth</option>,
+        <option>syslog</option>, <option>lpr</option>,
+        <option>news</option>, <option>uucp</option>,
+        <option>cron</option>, <option>authpriv</option>,
+        <option>ftp</option>, <option>local0</option>,
+        <option>local1</option>, <option>local2</option>,
+        <option>local3</option>, <option>local4</option>,
+        <option>local5</option>, <option>local6</option> or
+        <option>local7</option>. See
+        <citerefentry project='man-pages'><refentrytitle>syslog</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+        for details. This option is only useful when
+        <varname>StandardOutput=</varname> or
+        <varname>StandardError=</varname> are set to
+        <option>syslog</option>. Defaults to unset.</para>
+
+        <para>
+          Example:
+          <varname>FacilitiesStore=kern cron </varname> means only save LOG_KERN and LOG_CRON to disk,
+          drop messages from all others. Or, <literal>~</literal> syntax: <varname>FacilitiesStore=~kern daemon</varname>
+          implies store log messages of all facilities to disk, except for LOG_KERN and LOG_DAEMON</para>
```

Comment:
> https://github.com/pulseaudio/pulseaudio/blob/2417305ae755cbb3a92ca43a058f550809069cd9/src/pulsecore/log.c#L510. Some more daemons could be found in #5640.

This is why some other log daemon fixes the message facility because that's missing by having a default facility like imjournal if they don't find it. 

>The answer would be too long, but in a nutshell _TRANSPORT=journal has nothing to do with 
_TRANSPORT=syslog. The same is true for _TRANSPORT=kernel and _TRANSPORT=audit

Sorry Can't follow you . what  does it difference . Do we treat these differently ?